### PR TITLE
update speedup and cancel to make room for EIP-1559

### PR DESCRIPTION
**Pull request by brad-decker**
Makes the positional arguments of createCancelTransaction and createSpeedUpTransaction an option bag so that other fields (maxFeePerGas, etc) can be included in future PRs, this also updates configuring new gas properties so that when EIP-1559 transactions roll in they will be handled in the background. Right now in this PR if a user were to create a EIP-1559 transaction and then attempt to speed it up with a custom gas value this would not be used. Instead a 1.10 multiplier would be applied to both gas fields. 

### Manual Testing Steps
1. Make sure SHOW_EIP_1559_UI is false/0
2. Submit a transaction with low gasPrice (< 0.5) on any testnet
3. Attempt to cancel the transaction, and see that the suggested gasPrice is 10% higher than the original.
4. Attempt to speed up the cancellation and see that the gasPrice is 10% higher than the cancel.
5. Check transaction details for the proper history of resubmitting

Repeat the steps with a plain speed up and no cancel.

### CODE REVIEW

File: ui/components/app/gas-customization/gas-modal-page-container/gas-modal-page-container.container.js [hunk 1]
```diff
@@ -264,7 +264,8 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
       if (ownProps.onSubmit) {
         dispatchHideSidebar();
         dispatchCancelAndClose();
-        ownProps.onSubmit(gasLimit, gasPrice);
+        console.log('here');
```
Comment (darkwing) [inline ui/components/app/gas-customization/gas-modal-page-container/gas-modal-page-container.container.js:267]: Needs removal.

**Review by darkwing — CHANGES_REQUESTED** [review]
Outside of the console.log, this works as expected, and the code makes sense.  Would appreciate more eyes though.

File: ui/components/app/gas-customization/gas-modal-page-container/gas-modal-page-container.container.js (hunk 1 above)
Comment (brad-decker) [reply ui/components/app/gas-customization/gas-modal-page-container/gas-modal-page-container.container.js:267]:
```suggestion
```

**Review by darkwing — APPROVED** [review]

**Review by NiranjanaBinoy — APPROVED** [review]
LGTM

**Pull request merged**